esp/esp/program/views.py

```diff
@@ -312,13 +312,15 @@ def find_user(userstr):
         if cleaned.isnumeric() and len(cleaned) == 10:
             formatted = "%s%s%s-%s%s%s-%s%s%s%s" % tuple(cleaned)
             user_q = user_q | Q(contactinfo__phone_day=formatted) | Q(contactinfo__phone_cell=formatted)
-
+        #try name (including parent/emergency contact)
```

Comment:
Can you add a space after the # before the comment? And capitalize the first letter? :)